sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -1110,6 +1124,15 @@ static const struct platform_device_id board_ids[] = {
 					SOF_RT5682_MCLK_24MHZ |
 					SOF_RT5682_SSP_CODEC(0)),
 	},
+	{
+		.name = "mtl_mx98360_rt5682",
+		.driver_data = (kernel_ulong_t)(SOF_RT5682_MCLK_EN |
+					SOF_RT5682_SSP_CODEC(0) |
+					SOF_SPEAKER_AMP_PRESENT |
+					SOF_MAX98360A_SPEAKER_AMP_PRESENT |
+					SOF_RT5682_SSP_AMP(1) |
+					SOF_RT5682_NUM_HDMIDEV(4)),
+	},
```

Comment:
nit-pick: why do we have the two mtl platforms separated by jsl? Can we group MTL together, that would help reviewers long-term.

Reply:
Done